### PR TITLE
Create + User menus always line up with left side of buttons

### PR DESCRIPTION
**Pull request by KatieShipley**
Position the user_options and create_options menus relative to their respective buttons, so they are always left-aligned even as the User button width stretches from longer names.

## Links
- jira ticket: [PP-108](https://codedotorg.atlassian.net/browse/PP-108)

## Result
create_menu_fix

Edit: the dropdowns are now right-aligned instead of left-aligned, but will still always line up with their respective buttons.
right-align-dropdown

### CODE REVIEW

File: shared/css/user-menu.scss [hunk 1]
```diff
@@ -31,10 +32,19 @@
       
     }
   }
+
+  .user_options, .create_options {
+    left: 0px;
```
Comment (daynew) [inline shared/css/user-menu.scss]: Double check left-to-right languages.

**Review by daynew — APPROVED** [review]
Looks really good!

File: shared/css/user-menu.scss (hunk 1 above)
Comment (dju90) [reply shared/css/user-menu.scss]: You mean **right-to-left** languages, right? :)

Comment (KatieShipley) [reply shared/css/user-menu.scss]: Works the same in RTL, the position of the buttons in the header don't change :)

**Pull request merged**